units/check_blocked_tasks.go

```diff
@@ -126,6 +126,12 @@ func checkUnmarkedBlockingTasks(t *task.Task, dependencyCaches map[string]task.T
 
 	dependenciesMet, err := t.DependenciesMet(dependencyCaches)
 	if err != nil {
+		grip.Error(message.Fields{
+			"message":      "checking if dependencies met for task",
```

Comment:
if dependencies _are_ met for task 
might sound better

Reply:
this is meant to match the logging below i think, which we probably shouldn't change so that we can search consistently since this is still a problem